server/ladders-remote.ts

```diff
@@ -149,4 +173,34 @@ export class LadderStore {
 	static async visualizeAll(username: string) {
 		return [`<tr><td><strong>Please use the official client at play.pokemonshowdown.com</strong></td></tr>`];
 	}
+	/**
+	 * Calculates Elo based on a match result
+	 */
+	private calculateElo(previousUserElo: number, score: number, foeElo: number): number {
```

Comment:
This formula is wrong; it's generating different results from `ntbb-ladder.php`:

> Kennedy ⌐◼_◼'s rating: 1000 → 1064
> (+64 for winning)
> eaubot's rating: 1157 → 1128
> (-29 for losing)
> Kennedy ⌐◼_◼'s rating: 1000 → 1057
> (+57 for winning)
> eaubot's rating: 1157 → 1121
> (-36 for losing)

(gen8challengecup1v1 battle)

Reply:
Your duplicate result bug at least allowed us to catch this...